packages/fxa-graphql-api/src/auth/gql-customs.guard.ts

```diff
@@ -26,7 +26,7 @@ export class GqlCustomsGuard implements CanActivate {
     await this.customs.check({
       action: customsName,
       email,
-      ip: req.ip,
+      ip: req.ip ?? 'unknown', // req.ip may be undefined, but the original contract expects a string
```

Comment:
This doesn't seem to be dep upgrade related.

Reply:
Ah, I see that the type for that property has changed.

Well, we should not use a constant fallback value in this specific case, lest we want to group users/requests who do not have the same ip under the same "ip".  We can either try an empty string or update the type of the parameter for that `check` function.

Reply:
Empty string suggestion sounds good, thanks!  I added that.